hypothesis-python/src/hypothesis/extra/codemods.py

```diff
@@ -202,7 +202,17 @@ def leave_Call(self, original_node, updated_node):
         # Get the actual function object so that we can inspect the signature.
         # This does e.g. incur a dependency on Numpy to fix Numpy-dependent code,
         # but having a single source of truth about the signatures is worth it.
-        params = signature(get_fn(*qualnames)).parameters.values()
+        func = get_fn(*qualnames)
+        params = signature(func).parameters.values()
+        if len(updated_node.args) > len(params):
+            return updated_node
+
+        # st.floats() has a new allow_subnormal kwonly argument not at the end,
+        # so we do a bit more of a dance here.
+        params = signature(func).parameters.values()
```

Comment:
```suggestion
        params = signature(get_fn(*qualnames)).parameters.values()

        # st.floats() has a new allow_subnormal kwonly argument not at the end,
        # so we do a bit more of a dance here.
```

The existing `params` calculation is fine here, and we can just check for extra `args` after adjusting params for `floats()`.